Client/Teztez/Flows/Common/ActivitiesIntro/ViewModel/ActivitiesIntroViewModel.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  ActivitiesIntroViewModel.swift
+//  Teztez
+//
+//  Created by Adlet on 4/23/20.
+//  Copyright © 2020 crococoders. All rights reserved.
+//
+
+import UIKit
+
+enum ActivityIntroType {
+    case backward, blender, schulte, colorMatch
+}
+
+struct ActivitiesIntroViewModel {
```

Comment:
Split this into two viewModels:
- `ActivitiesIntroViewModel`
- `ActivitiesIntroBlockViewModel`

Reply:
If you want to handle all types of activities in advance create a factory that will emit viewModels. But I think it's not necessary at this point